portal-impl/test/unit/com/liferay/portal/test/rule/FeatureFlagTestRuleTest.java

```diff
@@ -35,4 +35,10 @@ public void testAnnotateMethod() throws Exception {
 		Assert.assertTrue(FeatureFlagManagerUtil.isEnabled("METHOD-123"));
 	}
 
+	@FeatureFlags(enable = false, value = "DISABLED-123")
+	@Test
+	public void testDisableFeatureFlag() {
+		Assert.assertFalse(FeatureFlagManagerUtil.isEnabled("DISABLED-123"));
```

Comment:
Since DISABLED-123 is not an actual feature flag, if I tried to check it I'd get `false`. So, to actually test this, I'd recommend doing a `setup` method that would set the feature flag to true, the this test would assert that it's false

Reply:
@thiago-buarqque @smottal
True, I did not think that through, please check it now.

The reason why I used BeforeClass annotation and not just Before, because it did not behave the same way for me when I was testing it and it worked appropriately for me only with the former annotation.